enginetest/queries/procedure_queries.go

```diff
@@ -1429,3 +1429,26 @@ var ProcedureShowCreate = []ScriptTest{
 		},
 	},
 }
+
+var NoDbProcedureTests = []ScriptTestAssertion{
```

Comment:
If you modeled this as a `[]ScriptTest` (instead of `[]ScriptTestAssertion`), then you could simplify the TestStoredProcedures function a little bit.

Reply:
See my comment below.
Summary: this is the simplest approach to running tests without a DB selected, possibly due to a bug within our test harness.